final/src/components/Fields/Dropdown.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Dropdown } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+
+export const DropdownField = ({ 
+  placeholder, 
+  option, 
+  input, 
+  type, 
+  onSubmit,
+}) => {
+  return (
+    <Dropdown
+      { ...input }
+      onChange={ (param, data) => {
+        input.onChange(data.value);
+        if (onSubmit) {
+          onSubmit(data);
+        }
+      } }
+      type={ type }
+      placeholder={ placeholder } 
+      fluid
+      selection
+      options={ option }
+    />
+  );
+};
+
+const propTypes = {
```

Comment:
Wrongly specified prop types. For example, you have specified that options should be object but all components pass array instead. It's also not clear why all props are not marked as required since no default value specified for them and no checking for undefined values in props .